db/migrate/20141204185232_add_force_password_reset.rb

```diff
@@ -0,0 +1,9 @@
+class AddForcePasswordReset < ActiveRecord::Migration
+  def up
+    add_column :users, :force_password_reset, :boolean, :default => true
```

Comment:
So there are two conflicting comments here. @eLobato states it should be false, where as @pitr-ch would like to keep it as true but move it out of the migration into the model.

@eLobato comment is easy to do :smile: 

@pitr-ch how would I go about adding it to the User initialize? Is that in ```./app/models/user.rb``` or somewhere else?